tasks/debian.yml

```
@@ -75,7 +73,6 @@
 - name: Install LLNG
   apt:
     name: "{{ item }}"
```

Should you not remove item here?

```suggestion
    name:
```

---

yes indeed :)

---

fixed